.travis.yml

```diff
@@ -1,16 +1,21 @@
-sudo: false
+sudo: required # due to manual install of make 4.1
```

Comment:
Thanks Jonathan, wanted to dig into the docs tonight but you beat me to it. They claim "20-50s boot time" for a sudo VM which I find acceptable.